spec/Dock/Docker/Dns/DnsDockResolverSpec.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace spec\Dock\Docker\Dns;
+
+use Dock\Docker\Dns\ContainerAddressResolver;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class DnsDockResolverSpec extends ObjectBehavior
+{
+    function it_is_a_container_address_resolve()
+    {
+        $this->shouldImplement('Dock\Docker\Dns\ContainerAddressResolver');
+    }
+
+    function it_returns_the_resolution_with_just_the_image_name()
+    {
+        $this->getDnsByContainerNameAndImage('container', 'image')->shouldContain('image.docker');
```

Comment:
Nop, it's not checking if a string contains a string, but it the returned array contains this string :)